client/src/components/StatsList/StatButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Stat } from "../../types/card/card.types";
+
+interface StatButtonProps {
+  stat: Stat;
+  playerId: string;
+  playRound: (stat: Stat, playerId: string) => void;
```

Comment:
If you also add and "isEnabled: boolean" prop here then we can use it to disable the stats buttons between rounds when we still want the user to see the stats but don't want them to be clickable.